docs/mlir_interoperation.ipynb

```diff
@@ -283,22 +277,23 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 7,
+   "execution_count": 13,
    "id": "b228d9e4",
    "metadata": {},
    "outputs": [
     {
      "name": "stdout",
      "output_type": "stream",
      "text": [
-      "builtin.module() {\r\n",
-      "  %0 : !i32 = arith.constant() [\"value\" = 1 : !i32]\r\n",
-      "  %1 : !i32 = arith.constant() [\"value\" = 2 : !i32]\r\n",
-      "  %2 : !i32 = arith.addi(%0 : !i32, %1 : !i32)\r\n",
-      "  %3 : !i32 = arith.addi(%2 : !i32, %2 : !i32)\r\n",
-      "  vector.print(%3 : !i32)\r\n",
-      "}\r\n",
-      "\r\n"
+      "Traceback (most recent call last):\n",
```

Comment:
I don't think we want this output from jupyter...

Reply:
But the output is actually there 😅 The pipeline was failing beforehand because passes were hardcoded... Do we really want these kind of tests?

Reply:
Ah, I see that was the issue. IMO, we should remove this then.